man/systemd-ssh-generator.xml

```diff
@@ -129,6 +129,19 @@
 
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>ssh.ephemeral-authorized_keys-all</varname></term>
+
+        <listitem><para>This credential should be a text file provided as an extra
+        <filename>authorized_keys</filename> file for all users for incoming connections via the generated
```

Comment:
"credential should be a text file" is a strange phrase. I'd expect something like "This credential provides additional contents for the `authorized_keys` file …".

The beginning of the text is taken from the previous item, but it is not great there too.

Please split the description in two paragraphs: reference part that says what the feature does and a explanation/hint part that describes the intended usage.

Reply:
I agree with the second part about the paragraph splitting but not really the first part.  "provides additional contents for the authorized_keys file" makes it sound like it might be modifying that file somehow... I'm trying to convey that this is a completely separate file which is also used in addition to the customary one...

Reply:
How about something like "Provides additional public keys, given in the customary `authorized_keys` format, ..." or so?